changelog/update-10325-remove-60-day-survey

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Removed the admin note to take the 60 day survey.
```

Comment:
Optional - add a bit about why we are doing this

```suggestion
Remove the 60 day survey admin note, since it will be redundant after we add the reviews prompt. 
```